### DISCUSSION TITLE
Best web server to deploy flask app on windows

### DISCUSSION THREAD

**Discussion by daparekh** [pallets/flask#4440] [Q&A] [answered]
We have a simple flask app to expose a few REST API endpoint and the requirement is to deploy on windows server.
Please suggest the best web server to host the application for production environment?

**Comment by tachyondecay** [comment, marked as answer]
[Waitress](https://docs.pylonsproject.org/projects/waitress/en/latest/) is a production-quality WSGI server and works on Windows.

**Reply by daparekh** [reply]
Thanks for the reply.

Is waitress open source?
can we get an official documentation for waitress and why is it not supported under deployment options on flask website- https://flask.palletsprojects.com/en/2.0.x/deploying/

**Reply by tachyondecay** [reply]
I linked to the Waitress docs above. From there you can get to the GitHub repository and view their license information.

As for the Flask docs, no one has contributed a page about Waitress. Maybe you can do that if you end up using it. I suspect it will be similar to the other standalone WSGI servers.
